app/Http/Requests/WritePostRequest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use App\Rules\TagMinRule;
+
+class WritePostRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'title' => 'required|min:6|max:150|unique:posts,title',
+            'content' => 'required|max:5000',
+            'tags' => [
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.KeySpecified</br>**Severity**: error</br>**File**: app/Http/Requests/WritePostRequest.php</br>**Line**: 32</br>Key specified for array entry; first entry has no key